src/js/components/PostCard/PostCard.js

```diff
@@ -142,7 +144,12 @@ class PostCard extends React.Component<Props, State> {
             ))}
           </div>
         )}
-        <div className="d-flex flex-column flex-md-row justify-content-between p-2 border-bottom">
+        {!isNative && (
+          <div className="post-card__imported px-1 py-half rounded-sm text-sm alert-info font-weight-normal">
+            This post was imported from outside of Lightful. Link clicks are not tracked.
+          </div>
+        )}
+        <div className="d-flex flex-column flex-md-row justify-content-between px-2 pb-2 pt-1 border-bottom">
```

Comment:
You are changing the look of the post card that is not imported. If you display a post card without an alert or an imported message you'll see you don't have enough padding on the top different. You're missing 1em